karma.conf.js

```diff
@@ -0,0 +1,70 @@
+// Karma configuration
```

Comment:
No, actually we have our gulpfile, package.json all are in root directory that's why I putted karma config under root directory.